fpdf/fpdf.py

```diff
@@ -2854,7 +2849,7 @@ def _putTTfontwidths(self, font, maxUni):
 
             if cid > 255 and (cid not in subset or cid >= len(font["cw"])):
                 continue
-            width = font["cw"][cid]
+            width = _char_width(font, cid)
             if width == 0:
                 continue
             if width == 65535:
```

Comment:
Since a value of 65535 is explicitly checked, I suggest to work with the original value here and to omit this line patch. You can't hide `font["cw"]` anyway as it's accessed a few lines above.

By the way, here is also the check for zero that was interfering with my original idea.

Reply:
Well spotted!
I opted to keep the call to `_char_width` and removed the 2 `if`s below